lib/dal/src/component.rs

```diff
@@ -1242,9 +1243,6 @@ impl Component {
             ctx.workspace_snapshot()?
                 .add_node(NodeWeight::Component(new_component_node_weight))
                 .await?;
-            ctx.workspace_snapshot()?
-                .replace_references(component_idx)
-                .await?;
```

Comment:
Do we no longer need this?

Reply:
add_node decides if it needs to replace refs now